packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -46,6 +47,24 @@ const VdiColSr = connectStore(() => ({
   sr: createGetObject(),
 }))(({ sr }) => <Link to={`srs/${sr.id}`}>{sr.name_label}</Link>)
 
+const VdiColSrContainer = connectStore(() => {
+  const getSr = createGetObject((_, props) => props.id)
+  const getContainer = createGetObject(
+    createSelector(getSr, sr => sr.$container)
+  )
+
+  return (state, props) => ({
+    container: getContainer(state, props),
```

Comment:
FYI, you can simply `return { container: getContainer }` and `connectStore` will do the job for you.